salt/elife-dashboard/scheduler.sls

```diff
@@ -117,6 +117,49 @@ configure-{{ app.name }}-log:
         - require:
             - file: configure-{{ app.name }}
 
+#
+# backup
+# bit of a hack for article-scheduler
+
+# separate descriptor living in different config directory so it doesn't conflict with elife-dashboard ubr config
+ubr-{{ app.name }}-db-backup:
+    file.managed:
+        - name: /etc/ubr2/elife-article-scheduler-backup.yaml
+        - source: salt://elife-dashboard/config/etc-ubr2-elife-article-scheduler-backup.yaml
+        - makedirs: true
+        - template: jinja
+        - require:
+            - {{ app.name }}-db-exists
+
+# separate UBR config living alongside regular config
+alt-ubr-config:
+    file.managed:
+        - name: /opt/ubr/alt-app.cfg
```

Comment:
can we call this `elife-article-scheduler.cfg` or `elife-article-scheduler-app.cfg`?

Reply:
I can do that